src/main/java/org/dominokit/domino/ui/datepicker/DateBox.java

```diff
@@ -63,7 +63,11 @@ protected HTMLInputElement createElement(String type, String placeholder) {
     private void init() {
         this.pattern = this.datePicker.getDateTimeFormatInfo().dateFormatFull();
         this.datePicker.addDateSelectionHandler((date, dateTimeFormatInfo) -> setStringValue(date, dateTimeFormatInfo));
-        this.modalListener = evt -> modal.open();
+        this.modalListener = evt -> {
+        	  if(!super.isDisabled()) {
+        	    modal.open();
```

Comment:
Sorry to nag, but while this switched to spaces, you are using 2 spaces here, but the convention is 4.

Also, space between `if` and `(` (also in the other file)